### ISSUE TITLE
Add grouping options for objects or elements

### ISSUE THREAD

**Issue by dhruvpatidar359** [dhruvpatidar359/nextdraw#8]
**Is your feature request related to a problem? Please describe.**
Currently we can only select only one element at once. 

**Describe the solution you'd like**
Create a sort of functionality to group select multiple elements at once.

**Additional context**
You have to create a new redux Slice to implement this , study the structure how we have implemented it for a single selected element and how it can be transformed to multiple elements.

**Comment by VanshikaSabharwal** [comment]
I can  work on it. Please assign this to me @dhruvpatidar359 . I am a gssoc contributor

**Comment by dhruvpatidar359** [comment]
> I can work on it. Please assign this to me @dhruvpatidar359 . I am a gssoc contributor

sure go ahead... 👩‍🚀

**Comment by VanshikaSabharwal** [comment]
@dhruvpatidar359  i am working on it. Hope you won't reassign it to anyone else.

**Comment by VanshikaSabharwal** [comment]
Hey @dhruvpatidar359 , when i am starting nextDraw  this loading bar is continuosly working and i am not able to see anything else on screen. Can you help me resolve this issue.
![Screenshot 2024-05-18 193142](https://github.com/dhruvpatidar359/nextdraw/assets/143436704/a0ade6ea-60de-4bf4-810b-f77861c1c545)

**Comment by dhruvpatidar359** [comment]
@VanshikaSabharwal  . Have you also installed the nextdrawbackend in this  ? 
Actaully what is happening , is this frontend is connected to the next draw backend . That nextDrawBackend is connected to MongoDb for getting the unique username. 

1 ) Two Options are there , first you can comment the code for the unique username
2 ) Or you have to  setup the backend and the mongoDb.

https://github.com/dhruvpatidar359/nextdraw?tab=readme-ov-file#installationlocal-setup

Have a look here , if still it comes , ping me . 👩‍🚀

**Comment by dhruvpatidar359** [comment]
![image](https://github.com/dhruvpatidar359/nextdraw/assets/103873587/0c6332d2-7208-4668-bb08-68ecf29d6554)

this is ,i am taking about

**Comment by VanshikaSabharwal** [comment]
Yes, i got it . I have commented the code and now the web app works fine. Thank you @dhruvpatidar359
